tweets-service/service.yml

```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
 
 messages:
   receives:
-    - tweets.create
-    - tweets.list
-    - tweets.get-details
-    - tweets.update
-    - tweets.delete
+    - create tweets
+    - list tweets
+    - get tweets details
+    - update tweets
+    - delete tweets
   sends:
-    - tweets.created
-    - tweets.listed
-    - tweets.details
-    - tweets.not-found
-    - tweets.updated
-    - tweets.deleted
-    - tweets.not-created
+    - tweets created
+    - tweets listed
+    - tweets details
+    - tweets not found
```

Comment:
Does tweets-service look for a single tweet or multiple? If it's a single then this should be `tweet not found`